docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,128 @@
 # View
 
-The `View` class is the base class for all views. With this class you can manage the background color and images for UI components.
+The [View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) is the fundermental concept for the all components such as `Button`, `Image`, `Text`, etc.
```

Comment:
I used every `View` in the description as `View`. For the only few cases such as in the Table, in the Caption, or in the Hyper Link, I used View, because it looks better for me.
Some view in table is my mistake, I will change it View.

Reply:
Thank you.